recipes/avahi/all/conanfile.py

```diff
@@ -91,8 +91,13 @@ def package(self):
         rm(self, "*.la", os.path.join(self.package_folder, "lib"))
 
     def package_info(self):
+        self.cpp_info.set_property("cmake_target_name", "Avahi::Avahi")
```

Comment:
I disagree. While it could have been kept in cmake_find_package to not break users. This initial error should not be propagated to CMakeDeps.

Reply:
I understand from the [Consuming recipes](https://github.com/conan-io/conan-center-index/blob/master/docs/consuming_recipes.md#breaking-changes) docs that CCI cares nothing for backward compatibility for users of its hosted recipes, even when Conan has the mechanisms to support it (e.g., `cmake_target_aliases` in this case). It is a serious discouragement to contributors.